examples/bus_targets/main.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
+  required_version = ">= 0.14.0"
+}
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "named_event_mapping" {
+  source   = "../../"
+  bus_name = "the-knight-bus"
+  event_patterns = [
+    "event.DementorsAppear",
+    "event.UnderAgedMagicDone"
+  ]
+
+  targets = {
+    bus = [
+      "arn:aws:events:us-east-1:123456789012:event-bus/ministryOfMagic"
```

Comment:
if there isn't a harry potter faker data set, your use of it makes me feel like it would be an Internet Necessity :tm:

Reply:
i would love that so much